app/src/main/java/org/mozilla/tv/firefox/ext/String.kt

```diff
@@ -85,3 +85,6 @@ val String.isUriYouTubeTV: Boolean
 val String.isUriYouTubeTvVideo: Boolean
     get() = this.isUriYouTubeTV &&
             this.contains("/watch/")
+
+val String.isUriMozSignIn: Boolean
```

Comment:
I feel weird exposing this on all strings but I don't have a better solution right now. It was a little better for YouTube because we do have many branches for YouTube but I doubt we'll have a similar case here.

Reply:
Agreed, but I think consistency is the more important element here.  We can keep it in mind and try to work on a better solution.

Reply:
Agreed, but I think consistency is the more important element here.  We can keep it in mind and try to work on a better solution.